authorization/src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
-
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
I think this is what you added that makes it auto create the entities

Reply:
That makes a lot more sense, whenever I commented out the 'application.properties' it would not let me proceed and gave me errors.